test/functional/commands/keyboard-e2e-specs.js

```diff
@@ -42,6 +44,12 @@ async function runTextEditTest (driver, testText, keys = false) {
   return el;
 }
 
+async function clearKeyEvents (driver) {
```

Comment:
Will do. Essentially it just clears the view (the view just takes any key events coming in and prints their details out in a TextView). Previously we were clearing it by reloading the activity, but that is both inefficient and potentially hides issues with how the events are being processed (less so for key events, but I removed it in the interest of consistency).